Library/ViewModels/SelectCurrencyViewModel.swift

```diff
@@ -33,6 +39,51 @@ SelectCurrencyViewModelOutputs {
       initialChosenCurrency,
       self.didSelectCurrencySignal
     )
+
+    let updateCurrencyEvent = self.selectedCurrencyProperty.signal.skipNil()
+      .takeWhen(self.saveButtonTappedSignal.ignoreValues())
+      .switchMap { input in
+        AppEnvironment.current.apiService
+          .changeCurrency(input: ChangeCurrencyInput(chosenCurrency: input.rawValue))
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .map { _ in input }
```

Comment:
It ignores the response from the mutation which is empty and passes back the currency that was selected so that we know it was successfully updated as the selected currency.